test/.excludes-zjit/TestProc.rb

```diff
@@ -0,0 +1,6 @@
+exclude(/test_/, 'Tests make ZJIT panic')
+
+# exclude(:test_proc_args_pos_rest_block, 'Test crashes with ZJIT')
+# exclude(:test_proc_args_rest_post_block, 'Test crashes with ZJIT')
+
+# exclude(:test_binding_receiver, 'Test fails with ZJIT')
```

Comment:
As explained in description, I need to exclude all test methods to avoid the ZJIT panic. But I also identified these tests that'd also fail/crash even without the panic.